Assets/MixedRealityToolkit.SDK/Inspectors/Experimental/UX/ObjectManipulator/ObjectManipulatorInspector.cs

```diff
@@ -157,8 +157,8 @@ public override void OnInspectorGUI()
                     EditorGUILayout.LabelField(constraintName);
                     if (GUILayout.Button("Go to component"))
                     {
-                        Debug.Log($"Highlighting {ObjectNames.NicifyVariableName(constraintName)} (Script)");
                         Highlighter.Highlight("Inspector", $"{ObjectNames.NicifyVariableName(constraintName)} (Script)");
+                        EditorGUIUtility.ExitGUI();
```

Comment:
What does this do? I can't find it in Unity's docs.
Based on the name, it seems strange, since there's definitely more GUI code after it!

Reply:
It appears to actually be calling [GUIUtility.ExitGUI](https://docs.unity3d.com/ScriptReference/GUIUtility.ExitGUI.html). I'm not entirely sure what it does in this context, but it certainly doesn't hide or disable the GUI code after it.